From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Tue, 14 Feb 2012 21:55:51 +0000
Subject: spelling typo

---
 _AlgorithmsPlugins/SEDS/SEDS.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_AlgorithmsPlugins/SEDS/SEDS.cpp b/_AlgorithmsPlugins/SEDS/SEDS.cpp
index 932e2d9..1a0018a 100644
--- a/_AlgorithmsPlugins/SEDS/SEDS.cpp
+++ b/_AlgorithmsPlugins/SEDS/SEDS.cpp
@@ -1629,7 +1629,7 @@ bool SEDS::CheckConstraints(Matrix * A){
     }
 
     if (nCtrViolated == 0)
-        cout<<"Optimization finished succesfully!"<<endl;
+        cout<<"Optimization finished successfully!"<<endl;
     else{
         QMessageBox *qmes = new QMessageBox ();
         qmes->addButton("Ok",QMessageBox::AcceptRole);
