From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Tue, 14 Feb 2012 21:55:51 +0000
Subject: spelling typo

---
 _AlgorithmsPlugins/SEDS/SEDS.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/_AlgorithmsPlugins/SEDS/SEDS.cpp b/_AlgorithmsPlugins/SEDS/SEDS.cpp
index 4af2867..7d43779 100644
--- a/_AlgorithmsPlugins/SEDS/SEDS.cpp
+++ b/_AlgorithmsPlugins/SEDS/SEDS.cpp
@@ -1591,7 +1591,7 @@ bool SEDS::CheckConstraints(Matrix * A){
     }
 
     if (nCtrViolated == 0)
-        cout<<"Optimization finished succesfully!"<<endl;
+        cout<<"Optimization finished successfully!"<<endl;
     else{
         QMessageBox *qmes = new QMessageBox ();
         qmes->addButton("Ok",QMessageBox::AcceptRole);
-- 
